ext/liquid_c/liquid.c

```diff
@@ -36,6 +37,7 @@ void Init_liquid_c(void)
     id_to_s = rb_intern("to_s");
     id_call = rb_intern("call");
     id_compile_evaluate = rb_intern("compile_evaluate");
+    id_ivar_line_number = rb_intern("@line_number");
```

Comment:
Is there a reason `id_ivar_line_number` is declared in `liquid.c` rather than `block.c`? Is it/will it be used anywhere else?

Reply:
> this will let me re-use the same ID for re-using with `rb_attr_get`, which I will need for Liquid::Assign.compile in #96